driver/src/contract.rs

```diff
@@ -322,7 +322,10 @@ impl SnappContract for SnappContractImpl {
         new_state: H256,
         prices_and_volumes: Vec<u8>,
     ) -> Result<()> {
-        debug!("Applying Auction with result bytes: {:?}", &prices_and_volumes);
```

Comment:
I don't see that this was exceeding the line width (considering there are others added here that were longer). This seems like an unnecessary change.

Reply:
rustfmt breaks it for me whenever I save